src/meta/src/manager/metadata.rs

```diff
@@ -315,6 +315,14 @@ impl MetadataManager {
             .await
     }
 
+    pub async fn list_active_worker_node(
```

Comment:
Seems that we have a single usage. Can call `self.list_worker_node(worker_type, Some(State::Running)).await` directly instead of introducing a new method.